src/Components/FileExplorer.js

```diff
@@ -33,10 +33,10 @@ export default class FileExplorer extends Component {
     return (
       <LeftNav
         docked={false}
-        menuItems = {menuItems}
+      menuItems = {menuItems}
         ref='fileExplorer'
-        style = {leftNavStyle}
-        onChange = {this.handleSelectedFile.bind(this)}
+      style = {leftNavStyle}
+      onChange = {this.handleSelectedFile.bind(this)}
```

Comment:
ESLint error for props not being sorted alphabetically. And while you're at it you may as well correct that this should be a self-closing tag since it is empty.